build.gradle

```diff
@@ -77,10 +77,10 @@ subprojects {
 
     repositories {
         mavenLocal()
-        maven { url "http://repo.maven.apache.org/maven2" }
+        mavenCentral()
+        jcenter()
```

Comment:
Do you need `jcenter()`?
It might have been required because in the past DE was published to jcenter. 

Just checking, if it's required for other dependencies as well no problem.

Reply:
Good call, it seems to build just fine without it.